### PR TITLE
Tweak purge_url

### PR DESCRIPTION
**Pull request by jaap3**

### CODE REVIEW

**Comment by jaap3** [comment]
I've updated this PR so purging multiple urls will re-use the same `requests.Session`, which was my intention in the first place but I somehow forgot to actually do.

File: fastly/utils.py [hunk 1]
```diff
@@ -1,22 +1,20 @@
 import requests
 
+from urllib.parse import urljoin
+
 from django.conf import settings
 
 
-def purge_url(path):
+def purge_url(*paths):
```
Comment (berkerpeksag) [inline fastly/utils.py]: What do you think about renaming this to `purge_urls()`?

Comment (jaap3) [reply fastly/utils.py]: Done

**Review by berkerpeksag — APPROVED** [review]

**Comment by berkerpeksag** [comment]
Nice cleanup, thank you very much!

**Pull request merged**